src/main/java/it/pn/frontend/e2e/common/DettaglioNotificaSection.java

```diff
@@ -70,7 +70,7 @@ public void clickLinkAttestazioniOpponibile(int numeroLinkAttestazioniOpponibile
         if (attestazioniFile.get(numeroLinkAttestazioniOpponibile).isDisplayed()) {
             attestazioniFile.get(numeroLinkAttestazioniOpponibile).click();
         } else {
-            this.js().executeScript("arguments[0].scrollIntoView(true);", attestazioniFile.get(numeroLinkAttestazioniOpponibile));
+            js().executeScript("arguments[0].scrollIntoView(true);", attestazioniFile.get(numeroLinkAttestazioniOpponibile));
```

Comment:
Maybe you can actually use this:
https://github.com/pagopa/pn-frontend-e2e/blob/5b46832c7ab76e1ca9477cccf1c933698c363b9a/src/main/java/it/pn/frontend/e2e/common/BasePage.java#L46-L57

Reply:
this is an old method, a lot of tests that use its don't work